server/data_models/device.ts

```diff
@@ -8,9 +8,13 @@ type DeviceType =
 type DeviceStatus = 'Not fitted' | 'Fitted' | 'Removed'
 
 type Device = {
+  id: number
```

Comment:
Why do we have an additional field that isn't in the data dictionary schema?

Reply:
to reflect the same fields that exist in the API

Reply:
I guess my point is why are they in the API?